lib/ansible/modules/system/crypttab.py

```diff
@@ -194,7 +194,7 @@ def match(self, name):
     def __str__(self):
         lines = []
         for line in self._lines:
-            lines.append(str(line))
+            lines.append(str(line).splitlines()[0])
```

Comment:
It would probably make more sense to use `.rstrip("\n")` (or similar), and do it when reading from the file or processing it in the Line class than doing it here.